src/core/data.jl

```diff
@@ -738,3 +738,86 @@ function _nan2zero(b, a; val=0)
     b[and.(isnan.(b), a.==val)] .= 0.0
     return b
 end
+
+
+"Counts active ungrounded connections on edge components"
+function count_active_connections(data::Dict{String,<:Any})
```

Comment:
I'm a bit confused at what you mean by an 'active' connection. You ignore connections to a grounded terminal, so do you consider a connection to a grounded terminal to be 'inactive'?

When I read 'active', I would expect you to ignore the components that are disabled (status=DISABLED).

Reply:
what is your suggestion for a better name? in my mind active was just used to differentiate that I would not count connections that are only there because of the matrices/vectors being padded with zeros in many cases, which was causing confusion, and making it more difficult to evaluate whether the network was radial

Reply:
The rows (and columns for matrices) that are padded with zeros, correspond to connections to phases, which are not grounded, so this counting method would not remove those 'padded' connections. Perhaps if you describe your use case for this method, it would be easier for me to see its use and a good name.

Reply:
it is just a helper function to help a user quickly determine the number of _specified_ connections for power delivery so that a user can get some basic statistics about their network. And this method definitely accounts for the padding.